kolibri/core/assets/src/views/Navbar/index.vue

```diff
@@ -81,6 +82,10 @@
           return values.every(value => value.link.name);
         },
       },
+      textColor: {
```

Comment:
Similarly here, Navbar is _only_ used in AppBar, and is also removed from the public API in develop, so I think we can just reference theme variables directly.